src/EnergyPlus/SingleDuct.cc

```diff
@@ -6015,6 +6015,162 @@ namespace SingleDuct {
         }
     }
 
+    void setATMixerSizingProperties(int const &inletATMixerIndex, // index to ATMixer at inlet of zone equipment
+                                    int const &controlledZoneNum, // controlled zone number
+                                    int const &curZoneEqNum       // current zone equipment being simulated
```

Comment:
This might all work if curZoneEqNum was unique for every piece of zone equipment. But it's just the current controlledZoneNum, set [here in ZoneEquipmentManager::SimZoneEquipment](https://github.com/NREL/EnergyPlus/blob/develop/src/EnergyPlus/ZoneEquipmentManager.cc#L3475). So, unless I'm missing something, this only works if there's only one piece of zoneHVAC equipment in the zone. Seems like ZoneEqSizing in general could be messed up if there's more than one piece of zone equipment. Or does ZoneEqSizing get completely rewritten by each piece of zone equipment, getting re-used safely? I need to come up with a test file that tries to prove this.

If the other problems noted in the DOATo* files are fixed, I suppose this can go in, but I think we need a separate PR to change curZoneEqNum to be unique for each piece of zone equipment similar to CurTermUnitSizingNum.

Reply:
We'll have to think more about this. Currently, zone equipment (which don't use TU's) use the zone index to access FinalZoneSizing. So if there were 2 PTAC's in the same zone they would size the same unless one had an ATMixer and the other didn't. What may be needed is some proportional sizing method as is done with TU's (e.g., CurTermUnitSizingNum can include percentage of airflow or some such thing).

Reply:
Regarding the same heating capacity, these PTAC's use 0 flow rate for OA.

    ZoneHVAC:PackagedTerminalAirConditioner,
      SPACE1-1 PTAC,           !- Name
      0,                       !- Cooling Outdoor Air Flow Rate {m3/s}
      0,                       !- Heating Outdoor Air Flow Rate {m3/s}
      0,                       !- No Load Outdoor Air Flow Rate {m3/s}

and therefore use FinalZoneSizing.MinOA to size the heating coil capacity. In develop, MinOA = 0.05 ish (if I recall correctly, or maybe 0.005) and in this branch ATMixer flow = 0.05. And the heating coil inlet temp is the same in both cases, hence the same capacity. So this branch fixes the reporting of the heating coil inlet conditions.

Reply:
I am also concerned about the use of these inputs to adjust equipment flow rate.

    Sizing:Zone,
      SPACE1-1,                !- Zone or ZoneList Name
      Yes,                     !- Account for Dedicated Outdoor Air System
      ColdSupplyAir,           !- Dedicated Outdoor Air System Control Strategy
      13.8,                    !- Dedicated Outdoor Air Low Setpoint Temperature for Design {C}
      15.6;                    !- Dedicated Outdoor Air High Setpoint Temperature for Design {C}

The IO ref says:

``
Field: Account for Dedicated Outdoor Air System
This is a choice field with choices Yes or No. The default is No. Choosing Yes means that the zone sizing calculation will use the subsequent inputs to calculate the heat gain or loss (heat gains are positive, heat loss is negative) imposed on the zone by a Dedicated Outdoor Air System (DOAS). This heat gain is then added to the zone design heat gain for the zone and the zone design air flow rate is adjusted to meet the DOAS heat gain plus the zone design heat gain.
``

This shouldn't be needed if the coil inlet conditions are correctly calculated. With both air flow (I haven't found where this is done yet) and inlet conditions being adjusted, this could be double counting the impact/load due to DOAS.

Reply:
As I understand this, you need this adjustment if it's a supply side mixer, because that type of mixer does not change the zone equipment coil inlet conditions.  The ATMixed primary flow goes directly to the zone from the DOAS and impacts the zone load, so the zone equipment should meet a different load. If it's an inlet side mixer, then you don't want this, because the coil inlet conditions will already acount for the DOAS air from the mixer, so this would double-count that.

Reply:
I think the ATMixer was added after DOAS. So how would we know during sizing where the DOAS air is directed, to the zone or to the inlet of the HVAC system?

Reply:
@mjwitte I could probably check right here (nside setATMixerSizingProperties) to see if Sizing:Zone input for Account for Dedicated Outdoor Air System is set to Yes and issue a warning and fatal. Other than that not sure what else I could do to avoid double counting DOAS.

Reply:
We don't. As it stands now, the user has to understand that and know when to use this and when not to. Prior to this change, one should use the DOAS sizing inputs for any DOAS, whether it's direct or through an ATMixer (supply-side and inlet-side). Now, this change automatically makes the adjustment for inlet-side ATMixers, but not the other two. Some options to consider:
1. If there's a supply-side ATMixer, then adjust the base load/zone supply flow rate that this equipment is sizing against.
2. When there's an ATMixer, check the zone sizing DOAS inputs and throw warnings. If active for an inlet-side mixer, warn that it's double-counting. If not active for a supply-side mixer, warn that it's not being accounted for.
3. Ultimately, we'd like to move toward eliminating the zone sizing DOAS inputs and always figure it out automatically, but it's too late for that here.